index.html

```diff
@@ -30,17 +51,17 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
-            </div>
+                <div class="box bk1">Box 1</div>
+                <div class="box bk2">Box 2</div>
+                <div class="box bk3">Box 3</div>
+                <div class="box bk4">Box 4</div>
+                <div class="box bk5">Box 5</div>
+                <div class="box bk6">Box 6</div>
+                <div class="box bk7">Box 7</div>
+                <div class="box bk8">Box 8</div>
+                <div class="box bk9">Box 9</div>
+                <div class="box bk10">Box 10</div>
+  </div>
         
```

Comment:
Great use of more than one class name,  another css selector called the :nth-child would be another way to implement the styles in a faster way. https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child